client/src/app/[locale]/page.tsx

```diff
@@ -10,10 +10,17 @@ import InformationOn from "@/containers/home/information-on";
 import KeyFeaturesGrid from "@/containers/home/key-features-grid";
 import KeyFeaturesReport from "@/containers/home/key-features-report";
 
-export const metadata: Metadata = {
-  title: "AmazoniaForever360+",
-  description: "",
-};
+type Params = Promise<{ locale: string }>;
+
+export async function generateMetadata({ params }: { params: Params }) {
```

Comment:
```suggestion
export async function generateMetadata({ params }: { params: Params }): Promise<Metadata> {
```
I think you can add back the Metadata from next and type the return of `generateMetadata` to be sure that you can't add things that are not compliant with the Metadata from next. https://nextjs.org/docs/app/api-reference/functions/generate-metadata#generatemetadata-function